osu.Desktop/osu.Desktop.csproj

```diff
@@ -5,43 +5,30 @@
     <OutputType>WinExe</OutputType>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Company>ppy Pty Ltd</Company>
     <Description>click the circles. to the beat.</Description>
-    <Copyright>ppy Pty Ltd 2007-2017</Copyright>
     <AssemblyName>osu!</AssemblyName>
     <Title>osu!lazer</Title>
     <Product>osu!lazer</Product>
     <ApplicationIcon>lazer.ico</ApplicationIcon>
-    <Configurations>$(CONFIGURATIONS);VisualTests</Configurations>
-    <Version>0.0.0.0</Version>
-    <FileVersion>0.0.0.0</FileVersion>
```

Comment:
I don't think you want to remove these two versions - they're used during deploy.